### ISSUE TITLE
FeatureRequest: Pinephone: OMAPs: upgrade to ofono v1.22 

### ISSUE THREAD

**Issue by carlosgonz0** [maemo-leste/bugtracker#714]

**Comment by sicelo** [comment]
why 1.22? maemo leste uses 1.34

or you meant to write 2.2, which is the version in upstream ofono?

**Comment by carlosgonz0** [comment]
> why 1.22? maemo leste uses 1.34
> 
> or you meant to write 2.2, which is the version in upstream ofono?

Yes i was wrong, i mean ofono  version 2.2. As the Pinephone it is the only viable phone for maemo in USA.

**Comment by MerlijnWajer** [comment]
Yeah, we'll get to this eventually. I don't know if anything big changed in ofono that caused them to increase the version, of if they just decided to change the version.

**Comment by sicelo** [comment]
> Yeah, we'll get to this eventually. I don't know if anything big changed in ofono that caused them to increase the version, of if they just decided to change the version.

quite a few things changed, yes. i should try and cherry-pick the droid4 patches on 2.2 to see what happens. i've been working on ofono myself recently, for some N900-related issues (https://lore.kernel.org/ofono/20231125201451.224574-1-absicsz@gmail.com/T/#u)

**Comment by carlosgonz0** [comment]
Thank you @sicelo, can not wait for meego-ofono v2.2,
